### ISSUE TITLE
clay: remote `%x` validates mark against `%base` desk only

### ISSUE THREAD

**Issue by tinnus-napbus** [urbit/urbit#6043]
**Describe the bug**

You can't `%x` eg a remote `desk.docket-0` cos it only searches `%base` for the `%docket-0` mark

**To Reproduce**
Steps to reproduce the behaviour:
1. `%x` a remote `desk.docket-0`
2. fail

**Expected behaviour**
Should search originating desk for mark, not `%base`

**Notify maintainers**
@philipcmonk

**Comment by philipcmonk** [comment]
Yes, this is a problem.  A workaround is to make sure the foreign desk gets "fully downloaded" by requesting `%v` for that case, and then you can make a `%x` request.

**Comment by ohAitch** [comment]
That sure doesn't sound very referentially transparent, what if the two marks disagree? (Esp for something like conversion to text when mounting to unix)
